Stripe/STPAddCardViewController.m

```diff
@@ -224,7 +224,7 @@ - (void)updateAppearance {
 }
 
 - (UIStatusBarStyle)preferredStatusBarStyle {
-    return ([STPColorUtils colorIsBright:self.theme.primaryBackgroundColor] 
+    return ([STPColorUtils colorIsBright:self.theme.secondaryBackgroundColor]
```

Comment:
I think these should actually be `self.navigationBar.stp_theme.secondaryBackgroundColor` maybe? But there might be a timing issue on when that is available/set.